{"name": "User cannot login with invalid credentials and sees a correct error message", "status": "passed", "steps": [{"name": "Given Open Gettop acc page", "status": "passed", "start": 1658676625293, "stop": 1658676628158}, {"name": "When Click on Email field", "status": "passed", "start": 1658676628158, "stop": 1658676628221}, {"name": "When Enter email testtest.com", "status": "passed", "start": 1658676628222, "stop": 1658676628335}, {"name": "When Click on PW field", "status": "passed", "start": 1658676628335, "stop": 1658676628394}, {"name": "When Enter pw: a", "status": "passed", "start": 1658676628395, "stop": 1658676628449}, {"name": "Then Click on Login btn", "status": "passed", "start": 1658676628449, "stop": 1658676629064}, {"name": "Then Verify correct credentials Error: Unknown username. Check again or try your email address.", "status": "passed", "start": 1658676629064, "stop": 1658676629094}], "start": 1658676622477, "stop": 1658676629249, "uuid": "320ac23a-9531-4378-82d1-9c64253bda51", "historyId": "f2b81a21e52815590a9eac2d195ef873", "fullName": "Account Login Error Handling: User cannot login with invalid credentials and sees a correct error message", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Account Login Error Handling"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}